Core/LocalAdmin.cs

```diff
@@ -99,12 +99,16 @@ internal enum HeartbeatStatus : byte
 
     internal LocalAdmin()
     {
-        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+        //Initial Setup of Platform Information.
+        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)){
             _scpslExecutable = "SCPSL.exe";
-        else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+        }
+        else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux)){
             _scpslExecutable = "SCPSL.x86_64";
+        }
         else
         {
+            //This occurs if the platform is not a Linux or Windows Machine.
```

Comment:
1. Comments are no longer relevant.
2. For one line if statement content we don't use braces.
3. Can you use "OperatingSystem.is..." here, in other places in this file (around lines 600 and 870) and in other files (IO/Config.cs and IO/PathManager.cs) as well?